### ISSUE TITLE
Ledger transfer tokens broken

### ISSUE THREAD

**Issue by jimbo2778** [harmony-one/harmony#4054]
**Describe the bug**
With the Ledger nano s connected to staking.harmony.one I cannot transfer $one. I can collect staking rewards and make staking transactions only. Firmware version is 2.1.0.

I hit the "confirm and sign" button and the popup disappears. No prompt on the ledger to approve the tx and no message saying the transaction has been submitted to the network. The coins never move.

**To Reproduce**
1. Connect ledger to staking.harmony.one
2. Click transfer funds
3. Paste address and select amount, hit next
4. Hit confirm and sign.

**Expected behavior**
I expected the ledger to populate a series of prompts to approve the transaction. Once approved it should've said transaction submitted to the network on the staking portal. Then the popup should've said transaction approved and provided the transaction hash.

**Screenshots**

**Environment (please complete the following information):**
 - OS: windows 10
- Go environment [ ```go env``` ]

**Additional context**

**Comment by sophoah** [comment]
For information, alternatively using the chrome extension interfacing with the harmony app on the nano S works.

**Comment by jimbo2778** [comment]
Upon further testing this is only affecting the staking portal. I was able to connect to the chrome extension wallet and send one from my ledger. I was also able to send one from metamask while connected with the eth app on the ledger.

**Comment by sophoah** [comment]
@potvik @hypnagonia @gupadhyaya could you guys have a look at it ?

**Comment by jimbo2778** [comment]
So the chrome extension wallet is going to sunset in 15 days. This going to create a major issue for ledger users. There will be no way to access their one or delegate/undelegate.

**Comment by jchak1624** [comment]
I believe we are all screwed...

**Comment by jimbo2778** [comment]
> I believe we are all screwed...

For now we can log into chrome extension with the ledger then log into the staking portal with chrome extension using the ledger acct. Once chrome extension is gone, we're stuck. I've started to undelegate everything now so I can transfer it all before time runs out.

**Comment by jimbo2778** [comment]
Additional note. This does not seem to affect brave browser users, only chrome browser.

**Comment by jimbo2778** [comment]
Just got a chance to test it on brave browser on windows 10 pc. Same issue. The user that reported it worked for them was on a mac.

**Comment by DustyInterface** [comment]
> > I believe we are all screwed...
> 
> For now we can log into chrome extension with the ledger then log into the staking portal with chrome extension using the ledger acct. Once chrome extension is gone, we're stuck. I've started to undelegate everything now so I can transfer it all before time runs out.


Thanks for the workaround jimbo2778. I tried everything as well, and this seems to be the safest option. Another option would be to import your Ledger seed phrase into a Harmony hot wallet -- but this seems pretty risky assuming you have other cyrpto accounts on Ledger. For whatever reason, when you hook up Ledger to One's Chrome extension wallet, you are unable to export the Ledger-One private key... maybe this is standard security, idk.

We could also wait for Harmony to fix the issue between Ledger and the staking portal. However, if you needed your One for any reason, it would be stuck on your Ledger until the fix. 

So for now, it seems Ledger users have to undelegate funds after hooking up Ledger to the Chrome extension wallet and then send funds to a new account before time runs out on the extension wallet. (You can still undelegate with just Ledger App and the portal, but will be unable to send funds form that account). 

Ideally we would be able to import our Harmony account from Ledger into Metamask. But since Metamask is Eth-based, it starts its list of importable accounts at some random address that it uses for Eth, Binance, etc. I scrolled through several pages of importable accounts, and couldn't find the option to import my original staked-Ledger One account. I don't know enough about the blockchain, but one could guess that you might have to scroll infinitely through the accounts to find one that holds your Harmony funds. For whatever reason, Metamask doesn't point to the account that Ledger originally unlocked using the Harmony App provided by Ledger. 

Once again, you might be able to import the Ledger-One address into Metamask by using your global Ledger seed phrase, but as noted above that might expose your hardware seed phrase to the interweb (correct me if I'm wrong). As for importing the hardware private key, I'm not sure how to obtain one from the Ledger or Chrome apps. 

TLDR;

Safest bet is to activate a new Harmony account or two, or three, through Metamask - import Ledger function. Then, send your Ledger funds using the Chrome app to the new accounts. From there, it is a normal Metamask-Ledger experience. 

Hope this helps anyone with the same concerns! Please suggest any other workarounds if they exist!

**Comment by jimbo2778** [comment]
> Okay so any update here? I can't send my undelegated ONE from https://staking.harmony.one. Ledger S/X doesn't get the opportunity to sign the send transaction.

For right now you can log into the Harmony Chrome extension wallet with the ledger to transfer funds.

**Comment by sophoah** [comment]
@potvik is working on it

**Comment by ahiipsa** [comment]
fixed [here](https://github.com/harmony-one/staking-dashboard/pull/637)

**Comment by ahiipsa** [comment]
> Tried to test but there appears to be some overlay issue: https://imgur.com/O7pCZDt

what browser do you use?

**Comment by DavidWhicker** [comment]
> > Tried to test but there appears to be some overlay issue: https://imgur.com/O7pCZDt
> 
> what browser do you use?

Chrome ( Mac ) - First time I have seen this overlay issue.

**Comment by DustyInterface** [comment]
Transfers work now, (Brave, OSX, Ledger Nano -S).

Thanks, Harmony team!

**Comment by jimbo2778** [comment]
> @jimbo2778 could you check and let me know if everything is OK?

I have fully tested it on chrome with windows 10 OS and I can happily report it is fully functional for me.

**Comment by sophoah** [comment]
Hello all, i am closing that issue since fix has been pushed and we have successful test above. Thank you all !
